src/screens/Checkout.jsx

```diff
@@ -0,0 +1,122 @@
+import { TextInput, View, Text, TouchableOpacity, Image } from "react-native";
+import React, { useState, useEffect } from "react";
+import {
+  containers,
+  stylesCheckout,
+  titles,
+} from "../styles/CheckoutScreen/Screen_Checkout";
+import { Formik } from "formik";
+import { MaterialIcons } from "@expo/vector-icons";
+import { COLORS } from "../styles/colors";
+
+export default function Checkout() {
+  return (
+    <View style={containers.bgContainer}>
+      <Image
+        source={require("../resources/checkoutNavigator.png")}
+        style={{ marginTop: 40, alignSelf: "center" }}
+      />
+      <Text style={titles.maintitle}>Enter your address:</Text>
+      <Formik
+        initialValues={{
+          address: "",
+          city: "",
+          postalcode: "",
+          country: "",
+        }}
```

Comment:
What do you think about putting this information in an external file like you do with styles?

```suggestion
        initialValues={source.data}
```